### PR TITLE
Recipe App

### PR DESCRIPTION
**Pull request by kingqabil**
Followed the given UML Relationship Diagram.

Login page and registration page:

-Should be built with Devise.
Food list:

-Should display a list of food as in the wireframe.
-Should lead to the form that allows users to add new food.

Recipes list:

-Should display a list of recipes created by the logged-in user as in the wireframe.
-Should lead to recipe details.
-If the user is the owner of the recipe, should allow the user to delete it.

Public recipe list:

-Should display a list of all public recipes ordered by newest as in the wireframe.
-Should lead to recipe details.

Recipe details:

-If the recipe is public or the user is the owner of the recipe, should display the recipe details as in the wireframe.
-If the user is the owner of the recipe, should lead to the form that allows the user to add new food.
-General shopping list view:
-Should show the list of food that is missing for all recipes of the logged-in user.
-Should count the total food items and total price of the missing food.
-Made sure there are no N+1 queries happening.
-Created a navigation menu that allows users to open all of the pages you created.
-Added unit and integration testing with RSpec and Capybara.
-Used Rails RESTful design and router.
-Added styling with Bootstrap.
-Added authorization rules with the cancancan gem

### CODE REVIEW

File: spec/requests/inventories_spec.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+    it 'renders correct template' do
+      expect(response).to_not render_template('show')
+    end
+
+    it 'check for placeholder text' do
+      expect(response.body).to include('Inventories by')
+    end
+  end
+end
```
Comment (DeliceLydia) [inline spec/requests/inventories_spec.rb:26]: - [ ] Please fix failing tests
test

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there! Kudos on:
- An awesome UI
- Working on all features
- Good README



## Suggested changes
_Check the comments under the review._


### Keep Rocking!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: spec/requests/inventories_spec.rb (hunk 1 above)
Comment (nevisende) [reply spec/requests/inventories_spec.rb:26]: Thank you for your review. We've fixed it now.

**Review by rukundoeric — APPROVED** [review]
Hi @vic778,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by nevisende** [comment]
Thank you for your reviews :)

**Pull request closed without merging**